tests/common.go

```diff
@@ -684,6 +685,45 @@ func ValidatePDSDataServices(ctx *scheduler.Context, errChan ...*chan error) {
 	})
 }
 
+func IsPoolAddDiskSupported() (bool, error) {
+
+	DMthin, err := IsDMthin()
+	if err != nil {
+		return true, nil
+	}
+	if DMthin {
+		pxVersion, err := semver.NewVersion("3.1.0")
+		if err != nil {
+			return false, err
+		}
+		log.Infof("DMTHIN is enabled")
+		driverVersion, err := Inst().V.GetDriverVersion()
+		if err != nil {
+			return false, err
+		}
+		var new_trimmedVersion string
+		parts := strings.Split(driverVersion, "-")
+		trimmedVersion := strings.Split(parts[0], ".")
+		if len(trimmedVersion) > 3 {
+			new_trimmedVersion = strings.Join(trimmedVersion[:3], ".")
+		} else {
+			new_trimmedVersion = parts[0]
+		}
+		currentPxVersionOnCluster, err := semver.NewVersion(new_trimmedVersion)
+		if err != nil {
+			log.InfoD("[semver.NewVersion] error is", err)
+			return false, err
+		}
+		if currentPxVersionOnCluster.GreaterThan(pxVersion) {
+			err = fmt.Errorf("drive add to existing pool not supported for px-storev2 or px-cache pools ")
```

Comment:
How about we include new_trimmedVersion?

Reply:
How about we let the test case handle the error?

Reply:
logged the new version and we have did fmt.errorf because we dont want error that semver produces us as it may be misleading , so we have did error format